rust/psibase/src/boot.rs

```diff
@@ -276,10 +281,21 @@ pub fn js_create_boot_transactions(
     let expiration = TimePointSec::from(now_plus_120secs);
     let prod = js_err(ExactAccountNumber::from_str(&producer))?;
 
-    // Todo: Allow parameterization of compression level from browser
-    let compression_level = 4;
+    let initial_key = public_key_pem
+        .map(|k| -> Result<AnyPublicKey, JsValue> {
+            let data = pem::parse(k.trim()).map_err(|e| JsValue::from_str(&e.to_string()))?;
```

Comment:
Should check that `data.tag()` is `"PUBLIC KEY"` and that the contents can be parsed as a `SubjectPublicKeyInfo`.

Reply:
done

Reply:
James addressed this.